model/src/time.ts

```diff
@@ -9,4 +9,11 @@ export type Minutes = number & {};
 export interface Timestamp {
 	readonly seconds: number;
 	readonly nanoseconds: number;
+}
+
+// Fonction utilitaire pour créer un Timestamp à partir d'une Date
+export function fromDate(date: Date): Timestamp {
```

Comment:
This utilitary function might be more appropriate in the `Time` namespace in `utils/time.ts`.